ArjunaJTS/jts/tests/classes/com/hp/mwtests/ts/jts/orbspecific/resources/AtomicWorker3.java

```diff
@@ -74,20 +76,20 @@ public static void randomOperation (int thr, int level)
 		{
 		    current.begin();
 
-		    Util.indent(thr, level);
-		    System.out.println("begin");
+		    logger.info(""+level);
```

Comment:
the interleaving of the threads just did not work for me as the resulting out.print just interleaved. I purposefully didn't touch the other tests that do use Util.java as I didn't have an issue with those tests and wanted to minimize the change set.

I would therefore prefer to not make further changes for this one unless there is something you spotted?